utilities/sbStripSlugUrl.ts

```diff
@@ -31,7 +31,25 @@ export const sbStripSlugURL = (url:string): string => {
     return '';
   }
 
-  const parts = url.split('/');
+  // Remove the leading slash from the URL
+  const urlNoLeadingSlash = url.replace(/^\//, '');
+
+  /**
+   * If the URL is just the prefix, return a slash.
+   * Occasionally the cached_url for the momentum homepage still shows up as
+   * 'home' so adding the last check just in case.
```

Comment:
Where is `home` coming from? The `real_path`?

Reply:
I think it's somewhere in the cache 😬 It only happens sometimes in Storyblok (they have some weird caching issue when on their site occasionally). When I looked at the draft JSON, "home" was not the slug, but I've seen it rendered a few times in the editor so just putting it here just in case.

Reply:
Huh. Ok. Thanks for explaining.